delivery_methods.py

```diff
@@ -57,7 +57,8 @@ def get_locker_id():
                 f" ul.{chosen_machine["ulica"]} {chosen_machine["numer"]}"
             )
 
-            return {given_machine_id: chosen_machine}
+            chosen_machine["paczkomat ID"] = given_machine_id
```

Comment:
staraj sie trzymac klucze w snake casie, 
```suggestion
            chosen_machine["machine_id"] = given_machine_id
            or
             chosen_machine["id"] = given_machine_id
```